src/components/guide-tool/views/GuideToolOngoingView.tsx

```diff
@@ -50,6 +53,13 @@ export const GuideToolOngoingView: FC<GuideToolOngoingViewProps> = props =>
         TryVisitRoom(parser.requesterRoomId);
     });
 
+    useMessageEvent<GuideSessionMessageMessageEvent>(GuideSessionMessageMessageEvent, event =>
+    {
+        const parser = event.getParser();
+
+        addMessengerEntry({ id: -1, webId: parser.senderId, entityId: -1, name: userName, timestamp: ChatHistoryCurrentDate(), type: ChatEntryType.TYPE_IM, roomId: !roomSession ? 0 : roomSession.roomId, message: parser.chatMessage });
```

Comment:
Since we aren't sending any chatlogs we probably don't need to store the helper chatlog